cypress/integration/ContextMenu.spec.ts

```diff
@@ -6,7 +6,7 @@ describe('ContextMenu', () => {
     });
 
     describe('when using pointer', () => {
-      it.skip('should close open submenus and reopen the root menu when right clicking trigger', () => {
+      it('should close open submenus and reopen the root menu when right clicking trigger', () => {
```

Comment:
what happened here?

Reply:
We fixed the issue so the test can be re-introduced.